README.md
#Skillify


docs/changes.txt
Home:
Copy Change
⦁	Find Your success your way.
⦁	There should be a period at the end ^^^
⦁	The Impact
⦁	Make The Impact Background a light grey instead of black (this goes for The Impact page as well)
⦁	New Section 1:
⦁	Ready? (this should be centered)
⦁	All Students apply to and begin with the six month, Level 1 Program: “The Networker.”
***She also wants new text beside each 1-5 that shows that you should click on the button.
⦁	Button Text: Learn about Application and Program Tuition
⦁	Hover Text inside box:
⦁	All high school and college students are welcome to apply!  Once accepted, students can register for their Level 1 program.  The tuition for the six-month program is $490, and partial, need-based financial assistance is available to all families showing need.  Keep reading to learn about how the six-month program benefits students!

Section 2: Learn What It Takes
⦁	Button Text: Learn More About The Conference
⦁	New Copy: Before students can figure out what they’re interested in, they need to learn what it takes.  That’s why the six-month program kicks off with a one-day conference at the University of Southern California.  Students will learn the core professional skills that will leave them feeling empowered and ready to explore their interests beyond their school’s campus.

Section 3: Your Virtual Coach Has Your Back
⦁	Button Text: Learn More About Coaching
⦁	New Copy: Once students learn what it takes, it’s go-time! They will need help along the way, so each student will have a virtual coach who will be available based on the student’s schedule.  Coaches help students narrow down their interests, meet professionals in those fields, and support them in the way they need.

Section 4: Transform Conversations Into Opportunities
⦁	Button Text: Learn More About The Six Months Support
⦁	New Copy: There is a reason why the program is six months long.  We start with simple introductions to professionals.  Based on the student’s goal and schedule, the coach will provide one-on-one guidance on turning simple introductions into professional opportunities, like mentorships, job shadows, and internships.


Section 5: Cross It off Your List, You’re One Step Closer
⦁	THERE IS NO BUTTON OR HOVER TEXT FOR STEP 5, REMOVE IT
⦁	New Copy: These things take time. We give students time to build relationships, job shadow, and explore other interests if their initial interest didn’t work out. We’re a big fan of “Cross it off the list, you’re one step closer!” This isn’t about locking our students down, but empowering them to actively explore and find the best options for themselves.






About

Delete: “Meet the Team” and leave the text below
Note: At the very end in the copy where it says “Read our story below” move that to the bottom:
Skillify was founded in 2012 at the University of Southern California. The team consists of education enthusiasts and is supported by the USC faculty and social entrepreneurs.
Read Our Story Below

Advisors (updated titles)
Raaid Hossain: Advisor - Strategy
Daniel Fischer: Advisor - Technology
Tina Admans: Advisor - Social Responsibility
Patrick Henry: Advisor - USC
Luke Brown: Advisor - USC










The Impact

Change The Impact background to light grey to match the home page

Here is the order of Parents Testimonials

Thank you so much for this program! It has already made a strong impact in Daniella's life. She has clarified her career goal - event planner. She enrolled in a speed reading/summer reading program. She is also watching informative TedTalks, all of which bring maturity and personal growth! Just today she heard she got selected for an unpaid summer internship and is excited about working on a campaign to raise awareness and funds for Free Arts for Abused Children. She is doing all of this on her own and I could not be more proud of her.
- Patricia S., mother of senior at West Ranch High School

I think that this is a wonderful program to offer to high school students, especially at a time in our society when it can be SO hard to find work of any kind. This program really helps to empower our young people, and this is a huge relief to us parents, as well as to our children. Thank you
- Mitzi S., mother of sophomore at Ulysses S. Grant High School

Melissa absolutely loved Saturday! She said she never, not once, looked at the time - the day flew by. She has been talking about the program since Saturday, and has shown a real enthusiasm in continuing to participate. Thank you so much for this absolutely fabulous program! I know Melissa will continue to grow and thrive.
- Alex W., mother of freshman at Sherman Oaks Center for Enriched Studies

Spencer was very enthusiastic with what he learned at the conference and what he learned about himself. His confidence level rose and he understood the value of the tools he is being introduced to. He is formulating goals and feels very good about being able to accomplish them. Thank you very much for creating this valuable experience and for presenting the opportunity to Spencer in Mr. Kurz classroom.
- Dennis W., father of seniors at Mira Costa High School

My daughter is already empowered by having attended the conference. She came home feeling inspired and hopeful that there are more options out there for her than she thought there were.
- Marina M., mother of junior at Palisades Charter High School

We are so grateful Stephanie was chosen for this amazing opportunity. She is our oldest and we didn't get the chance to continue with college right after high school. She has worked so hard to live her dream of college and a career in Physical Therapy. Now with Skillify we know she is a step closer to making it a reality.
- Marcela R, mother of sophomore at West Covina High School

***At the end of each testimonial it says “mother of….” Etc. Include that in the copy and change the order. Don’t make them red, I just wanted you to see it.


---------------------------


Educators

www/js/script.js
$.fn.counter = function( options ) {
    var settings = $.extend({
        start:  0,
        end:    100,
        easing: 'swing',
        duration: 1000,
        complete: ''
    }, options );

    var el = $(this);

    $({count: settings.start}).animate({count: settings.end}, {
		duration: settings.duration,
		easing: settings.easing,
		step: function() {
			var mathCount = Math.ceil(this.count);
			el.text(mathCount);
		},
		complete: settings.complete
	});
};
function animate(){
	$('section:not(.animate)').each(function(){
		if( $(window).scrollTop() + $(window).height() * .67  >= $(this).position().top){
			$(this).addClass('animate').find('.counter').queue(function(){
				val = $(this).data('counter');
				$(this).counter({end: val, duration: 1000});
			});
		}
	});
};
$(window).load(function() {
    $('.page-loader').hide();
});
jQuery(document).ready(function($){
	animate();
    $('.content-wrap').css('margin-bottom', $('footer').height());
    $(window).scroll(function(){
		animate();
	});
    $('a.slider-arrow').click(function(event) {
        event.preventDefault();
        $('body').animate({ scrollTop: $('header').outerHeight() }, 1000);
    });
    $('.lightbox-trigger').click(function(event){
        event.preventDefault();
        $('.lightbox').hide();
        $(  "#" + $(this).data('lightbox') ).css('display', 'flex');
    });
    $('.lb-close').click(function(){
        $('.lightbox-content').hide();
    });
});


www/_php/contact.php
<?php /* Template Name: Contact Page Template */ ?>
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">

    <title>Contact | Skillify</title>

    <link rel="shortcut icon" href="http://www.skillifynow.com/img/favicon.png" type="image/x-icon">

    <!--[if lt IE 9]>
      <script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
      <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
    <![endif]-->
</head>
<body class="contact">
    <div class="page-loader">
        <div class="bar"></div>
    </div>
    <div class="page-wrap">
        <header style="position: relative;">
            <nav class="navbar navbar-default">
                <div class="container">
                    <div class="navbar-header">
                        <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#main-nav" aria-expanded="false">
                            <span class="icon-bar"></span>
                            <span class="icon-bar"></span>
                            <span class="icon-bar"></span>
                        </button>
                        <a class="navbar-brand" href="http://www.skillifynow.com/"><img src="http://www.skillifynow.com/img/logo.png" alt="Skillify"></a>
                    </div>

                    <div class="collapse navbar-collapse" id="main-nav">
                        <ul class="nav navbar-nav navbar-right">
                            <li><a href="http://www.skillifynow.com/">Home</a></li>
                            <li><a href="http://www.skillifynow.com/about">About</a></li>
                            <li><a href="http://www.skillifynow.com/impact">The Impact</a></li>
                            <li><a href="http://www.skillifynow.com/mentors">Mentors</a></li>
                            <li><a href="http://www.skillifynow.com/educator">Educators</a></li>
                            <li><a href="http://www.skillifynow.com/news">News</a></li>
                            <li><a href="http://www.skillifynow.com/contact">Contact</a></li>
                        </ul>
                    </div>
                </div>
            </nav>

            <div class="slider" id="Slider">
                <div class="slider-wrap">
                    <div class="slider-content text-center">
                        <div class="container">
                            <div class="row" style="margin-top: 100px; text-align: left;">
                                <div class="col-md-5">
                                    <h1>We would love to hear from you!</h2>
                                    <h5 class="normal">
                                        Have a question, comment, or concern?
                                        Please use the contact information listed below.
                                    </h5>
                                    <p style="line-height: 1.2em; font-size: 14px;">
                                        Email: <a href="mailto:contact@skillifynow.com" style="font-size: 14px;">contact@skillifynow.com</a>
                                        <br>Phone: <a href="tel:213-807-3335" style="font-size: 14px;">(213) 807-3335</a>
                                    </p>
                                    <p style="line-height: 1em; font-size: 9px;">
                                        You may also fill out the contact form below and we will reach out to you within 24 hours.
                                    </p>
                                </div>
                                <div class="col-md-7">
                                    <form action="#">
                                        <div class="col-md-6">
                                            <label for="first_name">First Name:</label><br>
                                            <input type="text" name="first_name">
                                        </div>
                                        <div class="col-md-6">
                                            <label for="last_name">Last Name:</label><br>
                                            <input type="text" name="first_name">
                                        </div>
                                        <div class="col-md-12">
                                            <label for="f_email">Email:</label><br>
                                            <input type="email" name="f_email">
                                        </div>
                                        <div class="col-md-12">
                                            <label for="f_message">Your Message:</label><br>
                                            <textarea name="f_message" rows="8" cols="80"></textarea>
                                        </div>
                                        <div class="col-md-12 text-right">
                                            <input type="submit" name="f_submit" value="Send Message">
                                        </div>
                                    </form>
                                </div>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
        </header>

        <div class="content-wrap">
            <section id="Choice" class="section_six">
                <div class="container">
                    <div class="row">
                        <div class="col-md-6">
                            <h2>Student?</h2>
                            <a href="https://skillify.wufoo.com/forms/s18u6a2x0dcjxu4/" class="button">APPLY NOW</a>
                        </div>
                        <div class="col-md-6">
                            <h2>Educator?</h2>
                            <a href="#" class="button">CONTACT US</a>
                        </div>
                    </div>
                </div>
            </section>
        </div>

        <footer>
            <div class="container">
                <div class="row">
                    <div class="col-md-12">
                        <ul class="social-media-buttons">
                            <li><a href="https://www.facebook.com/Skillify/"><img src="http://www.skillifynow.com/img/facebook.png"></a></li>
                            <li><a href="https://twitter.com/skillify"><img src="http://www.skillifynow.com/img/twitter.png"></a></li>
                            <li><a href="https://www.instagram.com/skillifynow/"><img src="http://www.skillifynow.com/img/instagram.png"></a></li>
                        </ul>
                        <ul class="footer-nav">
                            <li><a href="http://www.skillifynow.com/">Home</a></li>
                            <li><a href="http://www.skillifynow.com/about">About</a></li>
                            <li><a href="http://www.skillifynow.com/mentors">Mentors</a></li>
                            <li><a href="http://www.skillifynow.com/educator">Educator</a></li>
                            <li><a href="http://www.skillifynow.com/impact">Impact</a></li>
                            <li><a href="http://www.skillifynow.com/news">News</a></li>
                            <li><a href="http://www.skillifynow.com/contact">Contact</a></li>
                        </ul>
                        <div class="copyright">&copy; 2017 All Rights Reserved. Skillify, Inc.</div>
                    </div>
                </div>
            </div>
        </footer>
    </div>

    <link href="https://fonts.googleapis.com/css?family=Rubik:400,900|Playfair+Display:900" rel="stylesheet">

    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
    <link rel="stylesheet" href="http://www.skillifynow.com/css/normalize.css">
    <link rel="stylesheet" href="http://www.skillifynow.com/style.css">
    <link rel="stylesheet" href="http://www.skillifynow.com/css/responsive.css" media="(max-width: 768px)">

    <script src="http://www.skillifynow.com/js/vendors/jquery.min.js"></script>
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
    <script src="http://www.skillifynow.com/js/script.js"></script>
</body>
</html>


www/_php/index.php
<?php /* Template Name: Home Page Template */ ?>
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">

    <title>Skillify</title>

    <link rel="shortcut icon" href="http://www.skillifynow.com/img/favicon.png" type="image/x-icon">

    <!--[if lt IE 9]>
      <script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
      <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
    <![endif]-->
</head>
<body class="home">
    <div class="page-loader">
        <div class="bar"></div>
    </div>
    <div class="page-wrap">
        <header>
            <nav class="navbar navbar-default">
                <div class="container">
                    <div class="navbar-header">
                        <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#main-nav" aria-expanded="false">
                            <span class="icon-bar"></span>
                            <span class="icon-bar"></span>
                            <span class="icon-bar"></span>
                        </button>
                        <a class="navbar-brand" href="index.html"><img src="http://www.skillifynow.com/img/logo.png" alt="Skillify"></a>
                    </div>

                    <div class="collapse navbar-collapse" id="main-nav">
                        <ul class="nav navbar-nav navbar-right">
                            <li><a href="http://www.skillifynow.com/">Home</a></li>
                            <li><a href="http://www.skillifynow.com/about">About</a></li>
                            <li><a href="http://www.skillifynow.com/impact">The Impact</a></li>
                            <li><a href="http://www.skillifynow.com/mentors">Mentors</a></li>
                            <li><a href="http://www.skillifynow.com/educator">Educators</a></li>
                            <li><a href="http://www.skillifynow.com/news">News</a></li>
                            <li><a href="http://www.skillifynow.com/contact">Contact</a></li>
                        </ul>
                    </div>
                </div>
            </nav>

            <div class="slider" id="Slider">
                <div class="slider-wrap">
                    <div class="slider-content text-center">
                        <h1>
                            Find Your Success,
                            <br>Your Way.
                        </h1>
                        <p class="big">
                            Whether you're a parent, an educator, or a student, scroll down to learn
                            <br>about how we transform students into young professionals.
                        </p>
                        <img src="http://www.skillifynow.com/img/slider-arrow.png">
                        <a href="#" class="slider-arrow"></a>
                    </div>
                </div>
            </div>
        </header>

        <div class="content-wrap">
            <section id="Educators" class="section_one">
                <div class="container text-center">
                    <div class="row">
                        <div class="col-md-12">
                            <img src="http://www.skillifynow.com/img/educators.png">
                        </div>
                    </div>
                </div>
            </section>

            <section id="Impact" class="section_one dark-bg">
                <div class="container">
                    <div class="row">
                        <div class="col-md-12">
                            <h2 class="section-title">
                                The Impact
                            </h2>
                        </div>
                        <div class="col-md-4">
                            <div class="card">
                                <div class="card-title"><span class=" counter" data-counter="98">00</span>%</div>
                                <div class="card-desc">
                                   of students say their achieved
                                    <br>their goal of securing
                                    <br>a mentorship
                                </div>
                            </div>
                        </div>
                        <div class="col-md-4">
                            <div class="card">
                                <div class="card-title"><span class=" counter" data-counter="92">00</span>%</div>
                                <div class="card-desc">
                                    Of students say they achieved
                                    <br>their goal of securing
                                    <br>a job shadow
                                </div>
                            </div>
                        </div>
                        <div class="col-md-4">
                            <div class="card">
                                <div class="card-title"><span class=" counter" data-counter="85">00</span>%</div>
                                <div class="card-desc">
                                    Of students say they achieved
                                    <br>their goal of securing
                                    <br>an internship
                                </div>
                            </div>
                        </div>
                    </div>
                </div>
            </section>

            <section class="section_two section_dot_after">
                <div class="container">
                    <div class="row">
                        <div class="col-md-12 text-center">
                            <div class="glow" style="margin: 0 0 1em;">1</div>
                            <h2>Ready?</h2>
                            <p style="font-size: 22px;">
                                All Students apply to and begin with the six month, Level 1 Program: “The Networker.”
                            </p>
                            <a href="#" data-lightbox="one" class="lightbox-trigger">Learn about Application and Program Tuition</a>
                        </div>
                    </div>
                </div>
            </section>
            <section class="section_three section_dot_after">
                <div class="container">
                    <div class="row">
                        <div class="col-md-6 col-md-push-6 slider-from-right">
                            <img src="http://www.skillifynow.com/img/home-learn-image.png">
                        </div>
                        <div class="col-md-6 col-md-pull-6 slider-from-left">
                            <div class="glow" style="margin: 1em 0;">2</div>
                            <h2>Learn what it takes</h2>
                            <p>
                                Before students can figure out what they’re interested in, they need to learn what it takes.  That’s why the six-month program kicks off with a one-day conference at the University of Southern California.  Students will learn the core professional skills that will leave them feeling empowered and ready to explore their interests beyond their school’s campus.
                            </p>
                            <a href="#" data-lightbox="two" class="lightbox-trigger">Learn More About The Conference</a>
                        </div>
                    </div>
                </div>
            </section>

            <section class="section_four section_dot_after">
                <div class="container">
                    <div class="row">
                        <div class="col-md-6 slider-from-left">
                            <img src="http://www.skillifynow.com/img/home-your-image.png" style="position: absolute; right: 0; max-width: initial;">
                        </div>
                        <div class="col-md-6 slider-from-right">
                            <div class="glow" style="margin: 4em 0 1em;">3</div>
                            <h2>
                                Your virtual coach
                                <br>has your back
                            </h2>
                            <p>
                                Once students learn what it takes, it’s go-time! They will need help along the way, so each student will have a virtual coach who will be available based on the student’s schedule.  Coaches help students narrow down their interests, meet professionals in those fields, and support them in the way they need.
                            </p>
                            <a href="#" data-lightbox="three" class="lightbox-trigger">Learn More About Coaching</a>
                        </div>
                    </div>
                </div>
            </section>

            <section class="section_dot_after section_five">
                <div class="container">
                    <div class="row">
                        <div class="col-md-6 col-md-push-6 slider-from-right">
                            <img src="http://www.skillifynow.com/img/home-people-image.png">
                        </div>
                        <div class="col-md-6 col-md-pull-6 slider-from-left">
                            <div class="glow" style="margin: 4em 0 1em;">4</div>
                            <h2>
                                Transform conversations
                                <br>into opportunities
                            </h2>
                            <p>
                                There is a reason why the program is six months long.  We start with simple introductions to professionals.  Based on the student’s goal and schedule, the coach will provide one-on-one guidance on turning simple introductions into professional opportunities, like mentorships, job shadows, and internships.
                            </p>
                            <a href="#" data-lightbox="four" class="lightbox-trigger">Learn More About The Six Months Support</a>
                        </div>
                    </div>
                </div>
            </section>

            <section id="StartOver" class="section_six">
                <div class="container">
                    <div class="row">
                        <div class="col-md-12 text-center">
                            <h2>Cross It off Your List, You’re One Step Closer</h2>
                            <p style="font-size: 22px;">
                                These things take time. We give students time to build relationships, job shadow, and explore other
                                <br>interests if their initial interest didn’t work out. We’re a big fan of “Cross it off the list, you’re one step closer!”
                                <br>This isn’t about locking our students down,<span class="stylize" style="color: #4C46B5;"> but empowering them to
                                <br> actively explore and find the best options for themselves.</span>
                            </p>
                        </div>
                    </div>
                </div>
            </section>

            <section id="Choice" class="section_seven">
                <div class="container">
                    <div class="row">
                        <div class="col-md-6 slider-from-left">
                            <h2>Student?</h2>
                            <a href="https://skillify.wufoo.com/forms/s18u6a2x0dcjxu4/" class="button">APPLY NOW</a>
                        </div>
                        <div class="col-md-6 slider-from-right">
                            <h2>Educator?</h2>
                            <a href="#" class="button">CONTACT US</a>
                        </div>
                    </div>
                </div>
            </section>
        </div>

        <footer>
            <div class="container">
                <div class="row">
                    <div class="col-md-12">
                        <ul class="social-media-buttons">
                            <li><a href="https://www.facebook.com/Skillify/"><img src="http://www.skillifynow.com/img/facebook.png"></a></li>
                            <li><a href="https://twitter.com/skillify"><img src="http://www.skillifynow.com/img/twitter.png"></a></li>
                            <li><a href="https://www.instagram.com/skillifynow/"><img src="http://www.skillifynow.com/img/instagram.png"></a></li>
                        </ul>
                        <ul class="footer-nav">
                            <li><a href="http://www.skillifynow.com/">Home</a></li>
                            <li><a href="http://www.skillifynow.com/about">About</a></li>
                            <li><a href="http://www.skillifynow.com/mentors">Mentors</a></li>
                            <li><a href="http://www.skillifynow.com/educator">Educator</a></li>
                            <li><a href="http://www.skillifynow.com/impact">Impact</a></li>
                            <li><a href="http://www.skillifynow.com/news">News</a></li>
                            <li><a href="http://www.skillifynow.com/contact">Contact</a></li>
                        </ul>
                        <div class="copyright">&copy; 2017 All Rights Reserved. Skillify, Inc.</div>
                    </div>
                </div>
            </div>
        </footer>
    </div>

    <div class="lightbox-content" id="one">
        All high school and college students are welcome to apply!  Once accepted, students can register for their Level 1 program.  The tuition for the six-month program is $490, and partial, need-based financial assistance is available to all families showing need.  Keep reading to learn about how the six-month program benefits students!
        <div class="lb-close">✕</div>
    </div>
    <div class="lightbox-content" id="two">
        <div>
            Once accepted into the Level 1 program, students must choose a conference date, which is the first day of their six month program.  The conference is in-person and hosted at USC on a weekend.   Students are able to choose a conference date that works for their schedule.
            <br><br>
            Conference Agenda:
            <ul>
                <li>Networking Workshop</li>
                <li>Networking Event</li>
                <li>Resume Workshop</li>
                <li>LinkedIn Workshop</li>
                <li>Six Month Program Discussion</li>
                <li>Interview Workshop</li>
            </ul>
        </div>
        <div class="lb-close">✕</div>
    </div>
    <div class="lightbox-content" id="three">
        Students meet their virtual coach at the conference, and within 10 days of their conference, students and coaches have their one-on-one video chats.  In these video chats, students and coaches get to know each other, set a goal for the six month program, and create weekly actions steps based on the student's schedule to reach that goal.
        <div class="lb-close">✕</div>
    </div>
    <div class="lightbox-content" id="four">
        The six months of support also allow students to request one-on-one help with other challenges they may be facing, like time management, choosing the best classes, college admissions, stress management, etc.
        <div class="lb-close">✕</div>
    </div>

    <link href="https://fonts.googleapis.com/css?family=Rubik:400,900|Playfair+Display:900" rel="stylesheet">

    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
    <link rel="stylesheet" href="http://www.skillifynow.com/css/normalize.css">
    <link rel="stylesheet" href="http://www.skillifynow.com/style.css">
    <link rel="stylesheet" href="http://www.skillifynow.com/css/responsive.css" media="(max-width: 768px)">

    <script src="http://www.skillifynow.com/js/vendors/jquery.min.js"></script>
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
    <script src="http://www.skillifynow.com/js/script.js"></script>
</body>
</html>
